metapool/sample_sheet.py

```diff
@@ -826,3 +824,149 @@ def sample_sheet_to_dataframe(sheet):
     out.drop(columns='Sample_Project', inplace=True)
     out.sort_values(by='sample_well', inplace=True)
     return out.set_index('sample_id')
+
+
+def contains_replicates(sheet):
+    """Returns True if samples in sheet include one or more replicates.
+
+    Parameters
+    ----------
+    sheet: sample_sheet.KLSampleSheet
+        Object from where to extract the data.
+
+    Returns
+    -------
+    bool
+        True if samples in sheet include one or more replicates.
+    """
+    if 'contains_replicates' in sheet.Bioinformatics.columns:
+        for index, row in sheet.Bioinformatics.iterrows():
+            if row['contains_replicates'].lower() == 'true':
+                return True
+
```

Comment:
This is confusing for me, is this checking for the column `contains_replicates` and that its value is 'true'? What would happen if is `false`? Is this actually necessary? Also, would this assume that all projects have replicates or just the first one?

Reply:
The idea was that it would return true if the sample-sheet contained the contains_replicates column and at least one of the samples had a value of True; otherwise it would skip processing. The idea was to also return False for legacy sample-sheets as well as without the column they obviously didn't need processing.

 I've removed it entirely because it wasn't being used - the legacy sheets and pre-prep files should now just be the degenerate case when demuxing.